python/tests/integration/completions/test_chat_completions.py

```diff
@@ -60,6 +64,12 @@
 except KeyError:
     ollama_setup = False
 
+anthropic_setup: bool = False
+try:
+    if os.environ["ANTHROPIC_API_KEY"] and os.environ["ANTHROPIC_CHAT_MODEL_ID"]:
```

Comment:
@alliscode could you see if we have/can get any anthropic keys and get those added?